src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,18 +3,92 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_RANGE = 10;
+    private static final int WRONG_RANGE = 11;
+    private static final String NULL_EXPRESSION = "null";
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
zero is default value for int data type